ksql-engine/src/main/java/io/confluent/ksql/function/Blacklist.java

```diff
@@ -60,6 +59,6 @@
 
   @Override
   public boolean test(final String resourceName) {
-    return blackList == null || resourceName.matches(blackList);
+    return resourceName.matches(blackList);
```

Comment:
out of interest, why are we using a regex for such simple matching? Can't we just dump each line into a `Set` and use `contains(resourceName)`???

Reply:
It is intended to do partial matches, i.e., the regex produced would be something like:
`^(?:java\.lang\.Process|java\.lang\.Runtime|javax)\.?.*$`
which will match anything starting with `java.lang.Process`, `java.lang.Runtime`, `javax`

Reply:
Ah, gotcha.